php/libraries/Utility.class.inc

```diff
@@ -180,6 +180,30 @@ class Utility
         return $list;
     }
 
+    /**
+     * Returns a list of consents in the database
+     *
+     * @return array an associative array(
+     *                  "ConsentTypeID" => array(['Name']  => "Name",
+     *                                           ['Label'] => "Label"))
+     */
+    static function getConsentList()
+    {
+        $DB =& Database::singleton();
+
```

Comment:
should be 
      $factory = NDB_Factory::singleton();
      $db      = $factory->database();

Reply:
@PapillonMcGill could you explain this to me please? I've seen both used and not sure what the difference is

Reply:
@PapillonMcGill i do see that the other functions in Utility follow the way you mentioned

Reply:
@zaliqarosli For more info, should ask @xlecours. I saw that he made a PR https://github.com/aces/Loris/pull/3507 for that. I just wanted to do all uniformly.
Suspect it as to do with uniformity to simplify testing